cs-content-generator-sources/src/main/java/io/cloudslang/tools/generator/entities/CsInput.java

```diff
@@ -15,11 +15,13 @@
 
 package io.cloudslang.tools.generator.entities;
 
+import lombok.Data;
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.HashMap;
 import java.util.Map;
 
+@Data
```

Comment:
@robert-sandor  Since the @Data annotation has been added, is the this.defaultValue = StringUtils.defaultIfEmpty(defaultValue, ""); impacted if it will be removed as in the other Java classes?

Reply:
@tethryus Because the contructor is custom, we'll leave it as is, and Lombok will not generate it separately.

Reply:
Good point. Nice catch. The PR can be merged, since the build has passed.